content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -412,6 +412,46 @@ forEach([
 When you use this approach, both the testing framework and CI Visibility can tell your tests apart.
 
 
+## Intelligent Test Runner
+
+<div class="alert alert-warning">Intelligent Test Runner is currently in closed beta.</div>
+
+Read more about Intelligent Test Runner here.
```

Comment:
I think in here we should link to a section exclusive to intelligent test runner